src/crypto/crypto_ec.cc

```diff
@@ -767,8 +767,8 @@ Maybe<void> ExportJWKEcKey(
   int degree_bytes =
     (degree_bits / CHAR_BIT) + (7 + (degree_bits % CHAR_BIT)) / 8;
 
-  BignumPointer x(BN_new());
-  BignumPointer y(BN_new());
+  auto x = BignumPointer::Create();
```

Comment:
Why create but not New like V8?

```suggestion
  auto x = BignumPointer::New();
```

Reply:
No reason really. Will change